doc/api/http.md

```diff
@@ -676,6 +676,62 @@ Removes a header that's already defined into headers object.
 request.removeHeader('Content-Type');
 ```
 
+### request.reusedSocket
+
+<!-- YAML
+added: v12.12.0
```

Comment:
I'm not sure what version should be put here.

Reply:
It should be `added: REPLACEME`. The next time a release is made, tooling will replace it with the correct version number.